export_as_latex.ml

```diff
@@ -17,7 +17,8 @@ let proof_to_latex implicit_exchange notations proof =
         ^ "\\newcommand*{\\zero}{0}\n"
         ^ "\\newcommand*{\\limp}{\\multimap}\n\n"
         ^ "\\newcommand*{\\hypv}[1]{\\hypo{\\vdash #1}}\n"
-        ^ "\\newcommand*{\\exv}[1]{\\infer{1}[\\ensuremath{\\mathit{ex}}]{\\vdash #1}}\n"
+        ^ "\\newcommand*{\\exv}[2]{\\infer{1}[\\ensuremath{\\mathit{ex}}]{\\vdash #2}}\n"
+        ^ "\\newcommand*{% \\exv}[2]{\\infer{1}[\\ensuremath{\\mathit{ex}_{#1}}]{\\vdash #2}}\n"
```

Comment:
are you sure of this line ? @olaure01

Reply:
It is a comment in LaTeX: it is there to show how to make the permutation appear.
- do not touch anything, the permutation does not appear
- uncomment this line and comment the previous one, permutation appears
- you want the permutation to appear another, adapt the code

Reply:
Does it work on your side ? On my side (and on preprod as well) I get :
```
Runaway argument?
{\newcommand *{\axv }[1]{\infer {0}[\ensuremath {\mathit {ax}}]{\vdash \ETC.
! Paragraph ended before \new@command was complete.
<to be read again> 
                   \par 
l.34 
     
!  ==> Fatal error occurred, no output PDF file produced!
```